sklearn/preprocessing/_data.py

```diff
@@ -3044,6 +3044,23 @@ def fit(self, X, y=None):
         return self
 
     def fit_transform(self, X, y=None):
+        """
+        Fit PowerTransformer to X, then transform X.
+
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features)
+            The data used to estimate the optimal transformation parameters
+            and to be transformed using a power transformation.
+
+        y : None
+            Ignored.
```

Comment:
I recall that we used more the following when recently merging documentation fixes.

```suggestion
        y : Ignored
            Not used, present for API consistency by convention.
```

Reply:
Yes it's true. It slipped my mind. Thanks for catching it.